pkgs/development/compilers/elm/packages/elm-test-rs.nix

```diff
@@ -0,0 +1,20 @@
+{ lib, curl, rustPlatform, fetchurl, openssl, stdenv, pkg-config, Security, darwin }:
+rustPlatform.buildRustPackage rec {
+  pname = "elm-test-rs";
+  version = "2.0";
+
+  src = fetchurl {
+    url = "https://github.com/mpizenberg/elm-test-rs/archive/v${version}.tar.gz";
+    sha256 = "sha256:1manr42w613r9vyji7pxx5gb08jcgkdxv29qqylrqlwxa8d5dcid";
+  };
+
+  nativeBuildInputs = [ pkg-config ];
```

Comment:
I'm not sure if this is necessary. Have you tried to compile without pkg-config? I think rust relies on pkg-config only when linking to c libs which I don't think is done in case of this binary.

Reply:
I did not. I can try without it.

Reply:
I don't know about pkg-config but I can confirm that elm-test-rs is compiled by default without any C dependency. You could have some if using openssl instead of rustls for the `ureq` dependency (https client)

Reply:
Indeed, it works without `pkg-config`. I will force-push to remove it.

Reply:
I removed it.